src/moin/scripts/migration/moin19/import19.py

```diff
@@ -204,7 +204,9 @@ def __init__(self, path, deleted_mode=DELETED_MODE_KEEP,
 
     def __iter__(self):
         pages_dir = os.path.join(self._path, 'pages')
-        for f in os.listdir(pages_dir):
+        pages = os.listdir(pages_dir)
+        pages.sort()
```

Comment:
there is also `sorted` which would make this:
```
pages = sorted(os.listdir(pages_dir))
```

One must be aware that this is sorting the encoded directory names, not the decoded page names, though.